site/content/docs/4.3/components/navs.md

```diff
@@ -14,12 +14,14 @@ The base `.nav` component is built with flexbox and provide a strong foundation
 
 {{< callout info >}}
 The base `.nav` component does not include any `.active` state. The following examples include the class, mainly to demonstrate that this particular class does not trigger any special styling.
+
+Regarding accessibility, the active state should also be indicated with the `aria-current` attribute — using the `page` value for current page, or `true` for current item in a set.
```

Comment:
"Regarding accessibility, the active state should also be indicated with" > "To convey the active state to assistive technologies, use"